### PR TITLE
Update repo user docs

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
This pull-request is made at the request of @rickoco to give the GitHub repos for CORE-V cores a more common "look and feel".

### CODE REVIEW

**Comment by DBees** [comment]
Style guidelines refer to an inactive repo in PULP which refers out to LowRISC. Seems to be rather distant from OpenHW

**Review by zarubaf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
 
-See [style-guidlines](https://github.com/pulp-platform/style-guidelines)
+Note that Contributors are required to be covered by an [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php).
+Contributors are encouraged, but not required, to be a [member](https://www.openhwgroup.org/membership/) of the OpenHW Group.
+
+## The Mechanics
+1. From GitHub: [fork](https://help.github.com/articles/fork-a-repo/) the [cva6](https://github.com/openhwgroup/cva6) repository
+2. Clone repository: `git clone https://github.com/[your_github_username]/cva6`
+3. Create your feature branch: `git checkout -b <my_branch>.`<br> Please uniquify your branch name.
+See the [Git Cheats](https://github.com/openhwgroup/core-v-verif/blob/master/GitCheats.md) for a useful nominclature.
```
Comment (ASintzoff) [inline CONTRIBUTING.md:11]: s/nominclature/nomenclature/

**Review by JeanRochCoulon — APPROVED** [review]

**Comment by MikeOpenHWGroup** [comment]
Comment from @DBees 
> Style guidelines refer to an inactive repo in PULP which refers out to LowRISC. Seems to be rather distant from OpenHW

I believe you are referring to the current head of the master branch.  This pull-request replaces the reference to the inactive PULP repo with a direct reference to the lowRISC styleguide.  This guide was formally adopted by the CTG for RTL coding so it's the right one to point at.

**Review by ASintzoff — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,32 +8,35 @@ It has configurable size, separate TLBs, a hardware PTW and branch-prediction (b
 
 ![](docs/_static/ariane_overview.png)
 
-## New Directory Structure:
-The directory structure has been changed to cleanly separate the [CVA6 RISC-V CPU](#cva6-risc-v-cpu) core from the COREV-APU [FPGA Emulation](#corev-apu-fpga-emulation).
+## Directory Structure:
+The directory structure separates the [CVA6 RISC-V CPU](#cva6-risc-v-cpu) core from the COREV-APU [FPGA Emulation](#corev-apu-fpga-emulation).
```
Comment (davideschiavone) [inline README.md:12]: CORE-V

**Review by davideschiavone — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (DBees) [reply README.md:12]: @MikeOpenHWGroup sound good

Comment (MikeOpenHWGroup) [reply README.md]: Good catch @davideschiavone.  Fixed.

**Review by davideschiavone — APPROVED** [review]

**Pull request merged**